src/world_interface/real_world_interface.cpp

```diff
@@ -121,6 +138,18 @@ void world_interface_init() {
 	initMotors();
 }
 
+std::shared_ptr<robot::base_motor> getMotor(robot::types::motorid_t motor) {
+	auto itr = motor_ptrs.find(motor);
+
+	if (itr == motor_ptrs.end()) {
+		// motor id not in map
+		return nullptr;
```

Comment:
Can we log an error here? We should know if some code is accessing motors that don't exist. Something like:
```c++
log(LOG_ERROR, "getMotor(): Unknown motor %d\n", static_cast<int>(motor));
```